### ISSUE TITLE
Would be nice to see and control all my devices.

### ISSUE THREAD

**Issue by ogoldberg** [KraigM/HomeBridgeController#53]
Or is it supposed to be doing that already and I'm experiencing an issue?

**Comment by KraigM** [comment]
What do you mean by all your devices?

**Comment by ogoldberg** [comment]
Lights, locks, etc.

**Comment by KraigM** [comment]
What I mean is are you talking about HomeKit devices or HomeBridge devices?

**Comment by ogoldberg** [comment]
I have a wink hub that I literally just finished setting up with homebridge (and had an awful time doing it, but that's not really relevant I guess). I would like to control all the lights and other devices that my wink hub controls through your app. Is this a realistic request? It would be great to actually have a mac app that does everything since I don't always want to use my phone for all that.

**Comment by AppleTechy** [comment]
If all you are running is Wink stuff there was a WinkWeb interface
developed for control on the Mac. I can try to find the Link to the project
if u are interested.?

**Comment by KraigM** [comment]
Ah ok so you are talking about HomeKit which right now is not available on Mac.  In theory, I could give you access to the same functionality of HomeKit but only for the HomeBridge devices, but that seems like it could easily become a security hole.  Hopefully we will get HomeKit for Mac this year (WWDC is a week from Tuesday so we prob know then).

**Comment by ogoldberg** [comment]
@AppleTechy thanks for that. I looked it up. Are you talking about http://austincondiff.com/winkapp/ ? Looks pretty cool. Unfortunately it only does lights though. Do you know of another one?

@KraigM great point about security. I guess I should be a bit more conscientious about that. Looking forward to seeing some interesting announcements come out of WWDC, although I'm feeling pessimistic about the "official" options that will be provided. I guess we'll see.

**Comment by KraigM** [comment]
BTW I just finished up the first rough version of complete initial install/setup of HomeBridge via the app.  There is an installer for Raspberry Pi (full install onto SD card with nice wizard tools right in the app) and one for the local Mac (including the necessary command line tools from Xcode).  But it has not been released yet 😞 

Just out of curiosity, what was the hardest part of the install/setup of homebridge/homebridge-wink?

**Comment by AppleTechy** [comment]
That's it the orginal version. Actually a user of homebridge @techyowl made
an updated version. https://github.com/techyowl/WinkWeb Which has control
over a lot more devices

**Comment by AppleTechy** [comment]
@Kraigm The issue is the Wink plugin is setup to use API 1 keys/commands
but Wink did an upgrade to server API 2 which isn't backwards compatible.
So new users either have to bother a friend's old API key or use a generic
android app one.

**Comment by ogoldberg** [comment]
@AppleTechy Cool. Thanks! I'll check it out. This looks pretty good too, once you figure out how to log in: https://winkathome.net/Home.aspx

@KraigM thanks for asking about the hardest part of the homebridge setup. I tried both windows 10 and ubuntu, and then went back to windows 10. There were numerous points really where it boiled down to having the right versions of everything. Node, npm, visual studio, openssh, and one or two other steps along the way all really tripped me up on versions. Every single one of them. It was awful. I also struggled with using my own API keys, and then reverted to the generic ones which ended up working fine. Then I was confused by the empty accessories array in the config.json file, so I deleted it, and that wasted a bunch of my time too. The error messages were almost never helpful. It was pretty bad. I wasted my whole day, basically. Would be nice to have a one-click installer to ease the process.
